03-react-components/src/message.jsx

```diff
@@ -0,0 +1,16 @@
+import { Headline } from './core-components/headline';
+import { Paragraph } from './core-components/paragraph';
+
+export class Message extends React.Component {
+  render () {
+    const { headline } = this.props;
+    const { text } = this.props;
```

Comment:
Keine Kritik, nur eine Frage: Gibt es einen speziellen Grund, warum Du das in zwei Zeilen geschrieben hast statt in einer? (Also ist ja völlig okay so, ist mir nur aufgefallen)

Reply:
Da hat mir `eslint fix` wieder mal einen Streich gespielt.

Ich wollte hier eigentlich mit expliziten Zuweisungen arbeiten und hab das nun auch so angepasst (inkl. dem Ausschalten der ESLint Regel).

Ich nutze zwar sonst sehr gerne destructuring, kann mir aber vorstellen dass das jemanden aus einer anderne Sprache wie Java erstmal verwirren könnte. Ich will in der Folge beide Optionen zeigen, es hier aber mal so stehene lassen.

Reply:
Alles klar 👍